Tests/Functional/Controller/ConnectControllerTest.php

```diff
@@ -48,29 +48,29 @@ public static function getKernelClass(): string
 
     public function testRegistration(): void
     {
-        $mockResponse = $this->prophesize(ResponseInterface::class);
-        $mockResponse->getBody()
+        $mockResponse = $this->createMock(ResponseInterface::class);
+        $mockResponse->method('getBody')
             ->willReturn(json_encode(['access_token' => 'valid-access-token']));
 
-        $httpClient = $this->prophesize(ClientInterface::class);
-        $httpClient->sendRequest(Argument::type(RequestInterface::class))
-            ->shouldBeCalled()
-            ->willReturn($mockResponse->reveal());
+        $httpClient = $this->createMock(ClientInterface::class);
+        $httpClient->method('sendRequest')
+            ->with($this->createMock(RequestInterface::class))
```

Comment:
is this needed?

Reply:
Maybe not but I have rewrote the prophecy version into same as mock